### ISSUE TITLE
How to override input 3x3 param for Encoders

### ISSUE THREAD

**Issue by jaideep11061982** [qubvel/segmentation_models.pytorch#75]
Hi,i see that it is defaulted to false for your encoders in param dict. How i override it while passing it to seg models ?

**Comment by qubvel** [comment]
Please provide reference link to code fragment

**Comment by jaideep11061982** [comment]
`model = FPN("se_resnext50_32x4d", encoder_weights='imagenet', classes=5, activation=None,dropout=0.3)`

I want to make use ot 3x3 input as True
Inside FPN it gets encoder,passes input 3x3 as False.
I tried overriding it by overriding the given methods in your class but it dint work i got the error while loading of state dict saying missing keys for 3x3 kernel layer at the beginning.

'''
import segmentation_models_pytorch as smp
from  segmentation_models_pytorch.base import *
from segmentation_models_pytorch.encoders import *


def get_encoder1(name, encoder_weights=None):
    #print(encoders[name]['params']['input_3x3'])
    encoders[name]['params']['input_3x3']=True
    Encoder = encoders[name]['encoder']
    encoder = Encoder(**encoders[name]['params'])
    encoder.out_shapes = encoders[name]['out_shapes']
    
    if encoder_weights is not None:
        settings = encoders[name]['pretrained_settings'][encoder_weights]
        encoder.load_state_dict(model_zoo.load_url(settings['url']))

    return encoder

class FPN1(EncoderDecoder):
    """FPN_ is a fully convolution neural network for image semantic segmentation
    Args:
        encoder_name: name of classification model (without last dense layers) used as feature
                extractor to build segmentation model.
        encoder_weights: one of ``None`` (random initialization), ``imagenet`` (pre-training on ImageNet).
        decoder_pyramid_channels: a number of convolution filters in Feature Pyramid of FPN_.
        decoder_segmentation_channels: a number of convolution filters in segmentation head of FPN_.
        classes: a number of classes for output (output shape - ``(batch, classes, h, w)``).
        dropout: spatial dropout rate in range (0, 1).
        activation: activation function used in ``.predict(x)`` method for inference.
            One of [``sigmoid``, ``softmax``, callable, None]
        final_upsampling: optional, final upsampling factor
            (default is 4 to preserve input -> output spatial shape identity)
    Returns:
        ``torch.nn.Module``: **FPN**
    .. _FPN:
        http://presentations.cocodataset.org/COCO17-Stuff-FAIR.pdf
    """

    def __init__(
            self,
            encoder_name='resnet34',
            encoder_weights='imagenet',
            decoder_pyramid_channels=256,
            decoder_segmentation_channels=128,
            classes=1,
            dropout=0.2,
            activation='sigmoid',
            final_upsampling=4,
    ):
        encoder = get_encoder1(
            encoder_name,
            encoder_weights=encoder_weights
        )

        decoder = FPNDecoder(
            encoder_channels=encoder.out_shapes,
            pyramid_channels=decoder_pyramid_channels,
            segmentation_channels=decoder_segmentation_channels,
            final_channels=classes,
            dropout=dropout,
            final_upsampling=final_upsampling,
        )

        super().__init__(encoder, decoder, activation)

'''

**Comment by qubvel** [comment]
In that case pretrained weights  will be incompatible, so this is not an option for se_resnext50.
Use senet154, it has 3x3 input by default
